front/app/containers/Admin/projects/project/analysis/InputPreview/LongFieldValue.tsx

```diff
@@ -27,7 +36,38 @@ const SelectOptionText = ({
   const option = options?.data.find(
     (option) => option.attributes.key === selectedOptionKey
   );
-  return option ? <T value={option.attributes.title_multiloc} /> : null;
+  return option ? (
+    <T value={option.attributes.title_multiloc} />
+  ) : (
+    <>No answer</>
+  );
+};
+
+const FilterToggleButton = ({ customFieldId, value }) => {
+  const filters = useAnalysisFilterParams();
+  const filterValue = filters[`input_custom_${customFieldId}`];
+  const isFilterSet = filterValue?.includes(value);
+
+  const handleToggleFilterOption = (customFieldId, customOptionKey) => () => {
+    const newFilterValue = xor(filterValue, [customOptionKey]);
+    updateSearchParams({
+      [`input_custom_${customFieldId}`]: newFilterValue.length
+        ? newFilterValue
+        : undefined,
+    });
+  };
+
+  return (
+    <Button
+      onClick={handleToggleFilterOption(customFieldId, value)}
+      icon={isFilterSet ? 'close' : 'filter-2'}
```

Comment:
You could use our `IconButton` instead, from the component library.

Reply:
In the end sticking with it, the IconButton doesn't support a background and I think it looks cleaner and nicely aligned with the text that way.